.gitignore

```diff
@@ -1,5 +1,330 @@
-﻿
-bin
-obj
-.vs
-packages
\ No newline at end of file
+﻿## Ignore Visual Studio temporary files, build results, and
```

Comment:
Im afraid we don't need this lot of things in here, it is literally not needed.

Reply:
git ignores are for keeping unnecessary files out of your git repo. I.e. FRFuel.csproj.user.